File: src/main/webapp/WEB-INF/js/myRoute/myRoute.js
//복붙해옴 수정해야함
function MyRoute(userNickname, regionName, themeName, title, content, writeDate, storeName, latitude, longitude, detailAddress, nowLoginId) {

	this.num = 0; //글번호
	this.userNickname = userNickname; //회원닉네임
	this.regionName = regionName; //지역명
	this.themeName = themeName; //테마명
	this.title = title;  //글제목
	this.content = content; //글내용
	this.writeDate = writeDate; //작성일자
	this.upCount = 0; //추천수
	this.readCount = 0; //조회수
	this.storeName = storeName; //상호명
	this.latitude = latitude; //위도
	this.longitude = longitude; //경도
	this.detailAddress = detailAddress; //상세주소
	this.bookmark = 0; //메인목록페이지의 즐겨찾기 확인여부 (0 or 1)
	this.nowLoginId = nowLoginId;//현재 로그인한 계정
}

File: src/main/java/com/phoenix/hiddenplace/domain/MyRoute.java
package com.phoenix.hiddenplace.domain;

public class MyRoute {

	private int routeNum;
	private String userId;
	private String title;
	private String content;
	private String writeDate;
	private int upCount;
	private int readCount;
	private String userNickName; 
	private String mrTitleImgURL;
	
	public int getRouteNum() {
		return routeNum;
	}
	
	public void setRouteNum(int routeNum) {
		this.routeNum = routeNum;
	}
	
	public String getUserId() {
		return userId;
	}
	
	public void setUserId(String userId) {
		this.userId = userId;
	}
	
	public String getTitle() {
		return title;
	}
	
	public void setTitle(String title) {
		this.title = title;
	}
	
	public String getContent() {
		return content;
	}
	
	public void setContent(String content) {
		this.content = content;
	}
	
	public String getWriteDate() {
		return writeDate;
	}
	
	public void setWriteDate(String writeDate) {
		this.writeDate = writeDate;
	}
	
	public int getUpCount() {
		return upCount;
	}
	
	public void setUpCount(int upCount) {
		this.upCount = upCount;
	}
	
	public int getReadCount() {
		return readCount;
	}
	
	public void setReadCount(int readCount) {
		this.readCount = readCount;
	}
	
	public String getUserNickName() {
		return userNickName;
	}
	
	public void setUserNickName(String userNickName) {
		this.userNickName = userNickName;
	}
	
	public String getMrTitleImgURL() {
		return mrTitleImgURL;
	}
	
	public void setMrTitleImgURL(String mrTitleImgURL) {
		this.mrTitleImgURL = mrTitleImgURL;
	}
	
}
